src/main/java/org/folio/service/organization/OrganizationStorageService.java

```diff
@@ -44,9 +46,11 @@ public class OrganizationStorageService extends BaseService implements Organizat
   @Override
   public CompletableFuture<Organization> createOrganization(Organization organization, Context context,
       Map<String, String> headers) {
+    logger.debug("createOrganization:: Trying to create organization with name: {}", organization.getName());
     HttpClientInterface client = getHttpClient(headers);
     CompletableFuture<Organization> future = new CompletableFuture<>();
     if (isSameAccountNumbers(organization)) {
+      logger.warn("crateOrganization:: Account number is not unique");
```

Comment:
can we add account number from organization for this log?

Reply:
I guess List<Account> accounts don't have to String method in the Account class
`isSameAccountNumbers(organization)` check account numbers from the organization based on the unique size and original size.
I can add the account numbers size

Reply:
ok I am adding organization name